modules/turnos/institucion.routes.ts

```diff
@@ -0,0 +1,20 @@
+import { institucion } from './institucion.schema';
+import { MongoQuery, ResourceBase } from '@andes/core';
+import { Auth } from '../../auth/auth.class';
+
+
+class InstitucionResource extends ResourceBase {
```

Comment:
si actualizas el paquete también podes agregar la interfaz al ResourceBase.

```suggestion
class InstitucionResource extends ResourceBase<IIntitucion> {
```

Reply:
agregado